Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -417,7 +417,7 @@ def get_issue_fields(issue_creating=False, **issue_args):
     if issue_args.get('assignee'):
         if not issue['fields'].get('assignee'):
             issue['fields']['assignee'] = {}
-        issue['fields']['assignee']['name'] = issue_args['assignee']
+        issue['fields']['assignee']['id'] = issue_args['assignee']
```

Comment:
better to keep the argument as an account name and in the code convert it to account ID.

Reply:
The endpoint in the API doc was marked as `EXPERIMENTAL` so I didn't want to depend on it.
![Screen Shot 2021-01-24 at 17 00 09](https://user-images.githubusercontent.com/21142167/105634223-ae112880-5e65-11eb-9d47-941fc220864c.png)